docs/example_python.md

```diff
@@ -1,30 +1,194 @@
+- Optimize that time for fast feedback
 
+Obviously, this is a silly example. But it can be a useful example to confirm that Tilt is working
+as expected in your environment.
+
+All the code is in this repo:
+
+[tilt-example-python](https://github.com/windmilleng/tilt-example-python)
+
+To skip straight to the fully optimized setup, go to this subdirectory:
+
+[Recommended Tiltfile](https://github.com/windmilleng/tilt-example-python/tree/master/2-live_update)
+
+## Step 0: The Simplest Deployment
+
+Our server is just seven lines long, and all it does is serve us an HTML page:
+
+```python
+from flask import Flask, render_template
+app = Flask(__name__)
+
+
+@app.route("/")
+def serve():
+    return render_template("index.html")
+
+
+if __name__ == "__main__":
+    app.run()
 ```
-git clone https://github.com/windmilleng/servantes
-cd servantes; tilt up
+
+To start this server on Kubernetes, we need 3 configs:
+
+1. A [Dockerfile](https://github.com/windmilleng/tilt-example-python/blob/master/0-base/Dockerfile) that builds the image
+2. A [Kubernetes deployment](https://github.com/windmilleng/tilt-example-python/blob/master/0-base/kubernetes.yaml) that runs the image
+3. And finally, a [Tiltfile](https://github.com/windmilleng/tilt-example-python/blob/master/0-base/Tiltfile) that ties them together:
+
+```python
+docker_build('example-python-image', '.')
+k8s_yaml('kubernetes.yaml')
+k8s_resource('example-python', port_forwards=5000)
 ```
 
-Each widget is implemented by a different microservice backend.
+The first line tells Tilt to build an image with the name `example-python-image`
+in the directory `.` (the current directory).
+
+The second line tells Tilt to load the Kubernetes
+[Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#creating-a-deployment)
+yaml. The image name in the `docker_build` call must match the container `image`
+reference in the `example-python` Deployment.
+
+The last line configures port-forwarding so that your server is
+reachable at http://localhost:5000/. The resource name in the `k8s_resource` call
+must match the Deployment's `metadata.name` in `kubernetes.yaml`.
+
+Try it! Run:
+
+```
+git clone https://github.com/windmilleng/tilt-example-python
+cd tilt-example-python/0-base
+tilt up
+```
+
+Tilt will open a browser showing the web UI, a unified view that shows you app
+status and logs. Your terminal will also turn into a status box if you'd like to
+watch your server come up there.
+
+When the server is ready, you will see the status icon turn green. The log pane will display some output from Flask, starting with:
+> Serving Flask app "app"
+
+[![The server is up! (Click the screenshot to see an interactive snapshot.)](assets/docimg/example-python-1-serverup.png)](https://cloud.tilt.dev/snapshot/AZSXjOYLDi6w9IoCnCI=){:.is-image}
+*The server is up! (Click the screenshot to see an interactive view.)*
+
+## Step 1: Let's Add Benchmark Trickery
```

Comment:
This section left me a bit lost because it described the resource that created the `start-time.txt` file, but it didn't tell me what was it that then read it to calculate the time elapsed. IMO it would be helpful to have a sentence or two to mention that `get_update_time_secs()` was added.

Reply:
changed -- better?

Reply:
Perfect, thanks!